docs/building-a-form/creating-a-form-config-file.md

```diff
@@ -1,286 +1,173 @@
 # Creating a form config file
 
-Your form is generated from a JSON Schema configuration file called `form.js`, along with a few other key configuration files. For information about the basic files you need to configure a form, see "Create required files" in the *US Forms System Getting Started Guide*.
+Your form is generated from a JSON Schema configuration file called `form.js`, along with a few other key configuration files.
 
-### In this guide
-
-- [Example `form.js` file](#example-form.js-file)
-- [About the `schema` and `uiSchema` objects](#about-the-schema-and-uischema-objects)
-- [Configuring `uiSchema` using rjsf options](#configuring-uischema-using-rjsf-options)
-- [Configuring `uiSchema` using US Form System options](#configuring-uischema-using-us-form-system-options)
-
-### Example `form.js` file
```

Comment:
This content moved to a separate file (https://github.com/usds/us-forms-system/blob/schema-uischema/docs/building-a-form/quick-start-example-formjs-file.md)